package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
+          "type": "boolean",
+          "default": true,
+          "description": "Specifes whether OmniSharp should be used for C# code formatting."
```

Comment:
I think we're in agreement. Almost all of the "omnisharp.XXX" would require the suffix because OmniSharp would need to be reloaded for the option to have an effect. The "csharp.XXX" options that I listed above don't need the suffix.

This is really fit and finish outside of the scope of your PR though. Are you comfortable with your change?